Xamarin.Forms.Platform.Android/AppCompat/NavigationPageRenderer.cs

```diff
@@ -300,17 +298,20 @@ protected override void OnLayout(bool changed, int l, int t, int r, int b)
 				AView child = GetChildAt(i);
 				bool isBar = JNIEnv.IsSameObject(child.Handle, bar.Handle);
 
-				if (ToolbarVisible)
+				if (isBar)
 				{
-					if (isBar)
+					if (ToolbarVisible)
 						bar.Layout(0, 0, r - l, barHeight);
 					else
-						child.Layout(0, barHeight + ContainerPadding, r, b);
+						bar.Layout(0, -1000, r, barHeight - 1000);
```

Comment:
@StephaneDelcroix It hides toolbar if `SetHideNavigationBar` is `false` in ContentPage.

`-1000` is not my idea, that was earlier. [NavigationPageRenderer.cs:313](https://github.com/xamarin/Xamarin.Forms/blob/master/Xamarin.Forms.Platform.Android/AppCompat/NavigationPageRenderer.cs#L313)